sdk/containerregistry/container-registry/swagger/containerregistry.json

```diff
@@ -1837,22 +1763,22 @@
       "properties": {
         "repositories": {
           "type": "array",
-          "x-ms-client-name": "Names",
           "description": "Repository names",
           "items": {
             "type": "string"
           }
+        },
+        "link": {
+          "type": "string"
         }
       },
       "x-accessibility": "internal",
       "example": {
-        "repositories": [
-          "production/alpine",
-          "testing/alpine"
-        ]
+        "repositories": ["production/alpine", "testing/alpine"]
       }
     },
     "DeletedRepository": {
+      "x-ms-client-name": "DeletedRepositoryResult",
```

Comment:
This should be `DeleteRepositoryResult`, since it's the result of the `DeleteRepository()` method.